logic.py

```diff
@@ -116,11 +122,20 @@ def get_paste_listing(config):
         return "ERROR", "Paste listing has been disabled by the " +\
             "administrator.", 503
 
+    b = config['b']
```

Comment:
Just for completeness, should we convert all `config['b'].e` occurrences to something like that, or not? I see you've only changed one of them.